benchmark/single-source/SortLargeExistentials.swift

```diff
@@ -0,0 +1,81 @@
+//===--- SortLettersInPlace.swift -----------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// This test is a variant of the SortLettersInPlace.
+
+import TestsUtils
+
+protocol LetterKind {
+  var value: String { get }
+  func lessthan(_ rhs: LetterKind) -> Bool
+}
+
+// A struct which exeeds the size of the existential inline buffer.
+struct Letter : LetterKind {
+  let value: String
+
+  var a: Int = 27
+  var b: Int = 27
+  var c: Int = 27
+  var d: Int = 27
```

Comment:
Gotcha! Could you add a comment that explains this?

Also [42](https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Answer_to_the_Ultimate_Question_of_Life.2C_the_Universe.2C_and_Everything_.2842.29) is much better number… 👨‍💻💁‍♂️😇